src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -457,12 +457,12 @@ class TableToolPage extends Component<Props, State> {
                                 <>
                                   <div>Generated permanent link:</div>
                                   <LinkContainer
-                                    url={`${window.location.href}/permalink/${permalinkId}`}
+                                    url={`${window.location.host}/data-tables/permalink/${permalinkId}`}
```

Comment:
any reason this cant just be `~/data-tables/permalink` ?

Reply:
Hey @jamescutts - after looking into this, it is because the LinkContainer component actually just outputs this link in a span, not an anchor tag.  Therefore it explicitly needs the host putting on there to show the entire URL correctly as text.